File: Heat/Migrations/201507270857483_VAT-single.vb
Imports System
Imports System.Data.Entity.Migrations
Imports Microsoft.VisualBasic

Namespace Migrations
    Public Partial Class VATsingle
        Inherits DbMigration
    
        Public Overrides Sub Up()
            AlterColumn("dbo.InvoiceRows", "VAT_Rate", Function(c) c.Single(nullable := False))
        End Sub
        
        Public Overrides Sub Down()
            AlterColumn("dbo.InvoiceRows", "VAT_Rate", Function(c) c.Decimal(nullable := False, precision := 18, scale := 2))
        End Sub
    End Class
End Namespace

File: Heat.ConvertedToC#/Migrations/201507270857483_VAT-single.cs
using Microsoft.VisualBasic;
using System;
using System.Collections;
using System.Collections.Generic;
using System.Data;
using System.Linq;
using System.Xml.Linq;
using System.Diagnostics;
using System.Collections.Specialized;
using System.Configuration;
using System.Text;
using System.Text.RegularExpressions;
using System.Web;
using System.Web.Caching;
using System.Web.Mvc;
using System.Web.Mvc.Ajax;
using System.Web.Mvc.Html;
using System.Web.Routing;
using System.Web.SessionState;
using System.Web.Security;
using System.Web.Profile;
using System.Web.UI;
using System.Web.UI.WebControls;
using System.Web.UI.WebControls.WebParts;
using System.Web.UI.HtmlControls;
using System.Data.Entity.Migrations;

namespace Heat.Migrations
{
	public partial class VATsingle : DbMigration
	{

		public override void Up()
		{
			// AlterColumn("dbo.InvoiceRows", "VAT_Rate", Function(c) c.Single(nullable := False))
		}

		public override void Down()
		{
			// AlterColumn("dbo.InvoiceRows", "VAT_Rate", Function(c) c.Decimal(nullable := False, precision := 18, scale := 2))
		}
	}
}
